common/gyro-dashboard.cpp

```diff
@@ -0,0 +1,54 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2024 Intel Corporation. All Rights Reserved.
+
+#include "motion-dashboard.h"
+
+using namespace rs2;
+using namespace rsutils::string;
+
+gyro_dashboard::gyro_dashboard( std::string name )
+    : motion_dashboard( name )
+{
+}
+
+void gyro_dashboard::process_frame( rs2::frame f )
+{
+    write_shared_data(
+        [&]()
+        {
+            if( f && f.is< rs2::motion_frame >()
+                && ( f.as< rs2::motion_frame >() ).get_profile().stream_type() == RS2_STREAM_GYRO )
+            {
+                double ts = glfwGetTime();
+                auto it = frame_to_time.find( static_cast<float>( f.get_profile().unique_id() ) );
+
+                if( ts - last_time > frame_rate && it != frame_to_time.end() )
+                {
+                    rs2::motion_frame frame = f.as< rs2::motion_frame >();
+
+                    x_value = frame.get_motion_data().x;
+                    y_value = frame.get_motion_data().y;
+                    z_value = frame.get_motion_data().z;
+                    n_value = std::sqrt( ( x_value * x_value ) + ( y_value * y_value ) + ( z_value * z_value ) );
```

Comment:
Is there a need to see `norm` by users? I don't think it has any meaning. Lets consult with Evgeni.

Reply:
It will be more useful to see all 3 axes at once, and even that I am not sure is needed.

Reply:
> It will be more useful to see all 3 axes at once, and even that I am not sure is needed.

I will check how to implement it. The common draw line function is built to display only one graph at a time.

Reply:
> Is there a need to see `norm` by users? I don't think it has any meaning. Lets consult with Evgeni.

HI @ev-mp. what do you think? Is the Norm vector giving any information to a user?

Reply:
As discussed, will keep `norm` might be useful for accel.
Will also try to draw all 3 axes at once, but might be only in next PR